main/src/ca/uwaterloo/flix/language/phase/OccurrenceAnalyzer.scala

```diff
@@ -172,6 +172,60 @@ object OccurrenceAnalyzer {
           val List(e1, e2) = es
           (OccurrenceAst.Expression.ScopeExit(e1, e2, tpe, purity, loc), o.increaseSizeByOne())
 
+        case AtomicOp.Is(sym) if sym.name == "Choice" =>
+          val List(e) = es
+          (OccurrenceAst.Expression.Is(sym, e, purity, loc), o.copy(defs = o.defs + (sym0 -> DontInline)).increaseSizeByOne())
+
+        case AtomicOp.Is(sym) =>
+          val List(e) = es
+          (OccurrenceAst.Expression.Is(sym, e, purity, loc), o.increaseSizeByOne())
+
+        case AtomicOp.Tag(sym) =>
```

Comment:
Possibly many of these can be collapsed?

Reply:
Yes, once I refactor `OccurrenceAst` :)